Boundary for SpatialSeries (#567)
NeurodataWithoutBorders/nwb-schema: .github/PULL_REQUEST_TEMPLATE.md

For all schema changes:
- [ ] Add release notes for the PR to `docs/format/source/format_release_notes.rst`.
- [ ] Have you included the relevant issue number using "Fix #XXX" notation where XXX is the issue number? By including "Fix #XXX" you allow GitHub to close issue #XXX when the PR is merged.

> - [ ] `hdmf-common-schema` needs to point to the latest release and not the latest of the `main ` branch.

Reviewer: ```suggestion
- [ ] Make sure that `hdmf-common-schema` points to the latest release and not the latest commit on the `main` branch.
```
